rpcs3/Emu/Memory/vm.cpp

```diff
@@ -355,6 +355,7 @@ namespace vm
 		if (!shm)
 		{
 			utils::memory_protect(g_base_addr + addr, size, utils::protection::rw);
+			utils::memory_protect(g_sudo_addr + addr, size, utils::protection::rw);
```

Comment:
It was supposed to always have rw protection in this case, without the necessity to set it every time. It's a bug in another place.